app/src/main/java/org/jellyfin/mobile/ApplicationModule.kt

```diff
@@ -17,18 +19,19 @@ const val PLAYER_EVENT_CHANNEL = "PlayerEventChannel"
 const val WEBAPP_FUNCTION_CHANNEL = "WebAppFunctionChannel"
 
 val applicationModule: Module = module {
-    single { AppPreferences(get()) }
+    single { AppPreferences(androidContext()) }
```

Comment:
Maybe even change it to `androidApplication()` to specifically request the applicationContext?